GVFS/GVFS.Common/NuGetUpgrade/NuGetUpgrader.cs

```diff
@@ -322,7 +324,7 @@ public override bool TryRunInstaller(InstallActionWrapper installActionWrapper,
                 InstallActionInfo currentInstallAction = null;
                 try
                 {
-                    string platformKey = InstallManifest.WindowsPlatformKey;
+                    string platformKey = "macOS";

```

Comment:
Move platformKey to GVFSPlatform?

(I am guessing the code is this way now just to get things working on the Mac)